cli/utilities/utils.py

```diff
@@ -199,3 +200,17 @@ def put_cephadm_ansible_playbook(
 
     node.upload_file(sudo=True, src=playbook, dst=dst)
     log.info(f"Uploaded playbook '{playbook}' to '{dst}' on node.")
+
+
+def get_builds_by_rhbuild(rhbuild):
+    """
+    Get RHCS and RHEL version from rhbuild
+
+    Args:
+        rhbuild (str): build version
+    """
+    match = re.search(RHBUILD_PATTERN, rhbuild)
```

Comment:
Vaibhav, to confirm, the RHBUIL_PATTERN expects the data to be in this format "5.3-rhel-8.7" correct? And the re.search in that case return, 
('5.3', 'rhel-8')

But as per our current approach, we are passing rhbuild value as  build value alone (e.g rhbuild 5.3)
So to accommodate this do we have to change the parameter passing as well?

If the idea is to get the rhel platform, can't we use the `--platform` value?

Please correct me if I got it wrong

Reply:
@pranavprakash20 yes, you are correct about regex pattern and `rhbuild` parameter accepts 5.3. But this `rhbuild` is used to get build details from receipe file which gives details same as  `RHBUIL_PATTERN` pattern.